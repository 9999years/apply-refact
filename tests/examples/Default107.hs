foo = qux (\x -> f x >>= g)
