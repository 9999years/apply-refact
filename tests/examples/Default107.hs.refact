[("tests/examples/Default107.hs:1:12: Warning: Use >=>\nFound:\n  \\ x -> f x >>= g\nWhy not:\n  f Control.Monad.>=> g\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 12, endLine = 1, endCol = 27}, subts = [("f",SrcSpan {startLine = 1, startCol = 18, endLine = 1, endCol = 19}),("g",SrcSpan {startLine = 1, startCol = 26, endLine = 1, endCol = 27})], orig = "f Control.Monad.>=> g"}])]
