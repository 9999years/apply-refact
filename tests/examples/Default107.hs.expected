foo = qux (f Control.Monad.>=> g)
